### ISSUE TITLE
pm.response missing info using Newman but not Postman

### ISSUE THREAD

**Issue by skylineApiTesting** [postmanlabs/newman#1879]
1. Newman Version 4.3.1
 2. OS details: Debian 8, Debian 3.16.51-3 (2017-12-13) x86_64 GNU/Linux
 3. Are you using Newman as a library, or via the CLI? CLI, reporting to Testrail
 3. Did you encounter this recently, or has this bug always been there: 
 4. Expected behaviour: pm.response object returns full object when the response is HTML (works using postman)
 5. Command / script used to run Newman: `TESTRAIL_TITLE="API Test Newman" newman run Test_Collection.json --insecure -e Environment.json -r testrail,cli | tee TestLog.txt`
 6. Sample collection, and auxiliary files (minus the sensitive details):
Offending assertion:
`pm.test("Status code is 200", function () {
    pm.expect(pm.response.code).to.eql(200);
});`
 7. Screenshots (if applicable):
pm.response console log using Postman:
![image](https://user-images.githubusercontent.com/42553731/52147968-548b1000-2636-11e9-92c3-c1843322dc31.png)
pm.response console log using Newman (note empty header and cookie arrays and no code, status, etc.):
![image](https://user-images.githubusercontent.com/42553731/52148013-68cf0d00-2636-11e9-986e-75d72a274244.png)

Please let me know if I should include any more details.

**Comment by codenirvana** [comment]
@skylineApiTesting Can you share a sample collection(minus the sensitive details) to reproduce this issue?

**Comment by skylineApiTesting** [comment]
Yes, sorry. 
The environment variables used in these tests are pretty self explanatory. All of the above tests fail but work when using the Postman runner. The error reported for these is "Test failed: expected { Object (id, _details, ...) } to have property 'code'", which is because the pm.response object just does not have 'code' as one of the fields.

[samplecollection.txt](https://github.com/postmanlabs/newman/files/2827902/samplecollection.txt)

**Comment by codenirvana** [comment]
@skylineApiTesting I am not able to reproduce this issue. Your collection and assertions look fine.

I don't feel this is an issue with Newman. For issues like this, we have an active user community forum that will help to solve further issues. Visit: https://community.getpostman.com/

**Comment by shamasis** [comment]
@codenirvana - I am seeing a blank array in the sample collection’s response property. What can lead to this?

Was this exported from the app? If so, which version?

cc @numaanashraf

**Comment by sallyransom** [comment]
@skylineApiTesting, did you ever figure out what the issue is? I've come across the same problem with a 200 OK test.

**Comment by codenirvana** [comment]
Closing this issue as we are not able to reproduce this internally and we haven't seen other users facing the same.

Please feel free to reopen if the issue persists or you can help us with the steps to reproduce this issue.

**Comment by JoeyJGarcia** [comment]
@skylineApiTesting, I wish there was some solution to this, we are getting the same result but it seems limited to just a few people.

**Comment by adesokanayo** [comment]
I have same issues running newman from Azure Devops, did anyone figure what the solution might be ?

**Comment by JoeyJGarcia** [comment]
I had since moved on to rewriting the wrapper I was using and I didn't have any problems.  I only saw one other person posting a message about this error so I figured is was something that was unique to the app that I was using (and written by someone else).  I would just make sure your environment variable (and globals) are getting set appropriately.  If they collections run in Postman, they should run in the runner too so the one variable in this equation that I kept looking at, was the environment variables.  In my new rewrite it all seems to work fine.  I am using the npm newman module not the cli.
